src/webui/webui.cpp

```diff
@@ -113,13 +113,13 @@ void WebUI::configure()
             }
             else
             {
-                const QString errorMsg = tr("Web UI: Unable to bind to IP: %1, port: %2. Reason: %3")
+                m_errorMsg = tr("Web UI: Unable to bind to IP: %1, port: %2. Reason: %3")
```

Comment:
`m_errorMsg` should be cleared at begin of configure attempt. It is expected that `errorMessage()` returns empty string when `isErrore()` is `false`.

Reply:
Ok, done.